fedimint-api/src/core/server.rs

```diff
@@ -51,6 +54,10 @@ pub trait IServerModule: Debug {
 
     fn as_any(&self) -> &dyn Any;
 
+    fn database_version(&self) -> DatabaseVersion;
```

Comment:
I wonder if we should expose this, or just give modules some functions to deal with it internally.

Reply:
I think exposing this forces developers to think about database migrations, but I'm ok with changing it too.

Reply:
Removed